AmazonChimeSDK/AmazonChimeSDK/internal/video/DefaultVideoClientController.swift

```diff
@@ -339,6 +340,10 @@ extension DefaultVideoClientController: VideoClientDelegate {
             }
         }
     }
+    
+    public func videoClientTurnURIsReceived(_ uris: [String]) -> [String] {
+        return uris.map{ self.configuration.urlRewriter($0) }
```

Comment:
Nit: Add a space before `{`: `return uris.map { self.configuration.urlRewriter($0) }`; or simply pass in `urlRewriter` as a param: `uris?.map(self.configuration.urlRewriter)`